pkg/services/govmomi/util.go

```diff
@@ -440,7 +440,15 @@ func waitForIPAddresses(
 					// spec's static IP addresses.
 					isStatic := false
 					for _, specIP := range deviceSpec.IPAddrs {
-						if discoveredIP == specIP {
+						// The static IP assigned to the VM is required in the CIDR format
+						ip, _, err := gonet.ParseCIDR(specIP)
+						if err != nil {
+							ctx.Logger.Info(
```

Comment:
If we do #853, getting an error here should be impossible

Reply:
Ack. I'll remove the err check.

Reply:
Well.... we haven't done #853 yet, so I'd leave it in for now.

Reply:
#853 is going to be merged today, so I'd rather just remove this

Reply:
The PR for this is in (#970) - should we update this to not check the error any more?

Reply:
Ack, will remove the error check.